react/README.md

```diff
@@ -1,36 +1,50 @@
-# Boxwise react client 
+# Readme
 
-This front-end project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This frontend project of Boxtribute was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## All available Create-React-Scripts
+## Table of Contents
 
-In the project directory, you can run:
+1. [Contribution Guidelines](../CONTRIBUTING.md)
+2. [Development Set-up](#development-set-up)
+   1. [Set-up pre-commit](#set-up-pre-commit)
+   2. [Install node and yarn](#install-node-and-yarn)
+   3. [Linting and Formatting in VSCode](#linting-and-formatting-in-vscode)
+3. [Note about yarn and Docker](#note-about-yarn-and-docker)
+4. [Testing][#testing]
 
-### `yarn start`
+## Development Set-Up
 
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Following the [general set-up steps](../README.md), here a few steps that make your live easier working on the frontend.
 
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
+### Set-up pre-commit
 
-### `yarn test`
+[Pre-commit](https://pre-commit.com/) enables us to run code quality checks, such as missing semicolons, trailing whitespace, and debug statements, before you are committing your code. We chose pre-commit since it enables us to run these checks for both frontend and backend in just one place.
+The downside is that you need python to be installed on your computer.
+Please check the [backend README}(../flask/README.md#set-up-pre-commit) to set it up.
```

Comment:
} should be a ] such that the link is correctly displayed

Reply:
done